### ISSUE TITLE
Ambiguous 'Pedestrian Crosswalk' preset

### ISSUE THREAD

**Issue by SelfishSeahorse** [openstreetmap/iD#4519]
The `crossing=zebra` tag, which the 'Pedestrian Crosswalk' preset (node/way) defaults to, is ambiguous, because in countries outside the UK, zebra crossings are also used on pedestrian crossings controlled by traffic signals. As there is already a 'Street Crossing' preset, which doesn't default to a crossing type, I think it would be best to remove the 'Pedestrian Crosswalk' preset.

**Comment by bhousel** [comment]
see #4316 and many others

tl;dr - when I made this preset, I picked the best tag available from the list in taginfo.  While the raw numbers of crossings have taken off, the relative "market share" of `uncontrolled` vs `traffic_signals` vs `zebra` have not changed much since I introduced the preset.  "Zebra" is widely recognized (even outside the UK) as a crossing with stripes or similar markings on the ground.

I think there is some budding interest in fixing the current mess of crossing tagging as local communities (particularly in Seattle and San Jose) are starting to get serious about accessibility mapping projects.  We'll do whatever we can from the iD side to support their efforts and welcome any serious proposal to replace the `crossing=*` values with a schema that better meets the needs of people doing real work on pedestrian mapping.
